docs/pages/components/tile.md

```diff
@@ -10,186 +10,637 @@ summary:
+        </div>
+        <div class="fd-tile__footer">
+            <span class="fd-tile__footer-text">Unit, Notification</span>
+        </div>
     </div>
 </div>
-{% endcapture %}
-{% include display-component.html component=tile %}
-
 <br>
-
-## Product Tile
-`fd-product-tile` can be used to display product information.
-
-{% capture tile %}
-<div class="fd-product-tile">
-    <div class="fd-product-tile__media" style="background-image: url('https://placeimg.com/400/400/nature');"></div>
-    <div class="fd-product-tile__content">
-        <h3 class="fd-product-tile__title">Default Product Tile</h3>
-        <p class="fd-product-tile__text">Tile Description</p>
+<br>
+<h4>Launch Tile (Logo)</h4>
+<br>
+<div class="docs-section-container">
+    <div tabindex="0" class="fd-tile fd-tile--launch">
+        <div class="fd-tile__header">
+            <div class="fd-tile__title">Launch Tile (Logo) Title Text</div>
+            <div class="fd-tile__subtitle">Launch Tile Subtitle</div>
+        </div>
+        <div class="fd-tile__content">
+            <img class="fd-tile__logo" src="//unpkg.com/fundamental-styles/dist/images/sap-logo.png" srcset="//unpkg.com/fundamental-styles/dist/images/sap-logo@2x.png 1x, //unpkg.com/fundamental-styles/dist/images/sap-logo@3x.png 2x, //unpkg.com/fundamental-styles/dist/images/sap-logo@4x.png 3x" alt="SAP">
+        </div>
+        <div class="fd-tile__footer">
+            <span class="fd-tile__footer-text">Footer Text</span>
+        </div>
     </div>
-</div>
-
-<div class="fd-product-tile" role="button">
-    <div class="fd-product-tile__media" style="background-image: url('https://placeimg.com/400/400/nature');"></div>
-    <div class="fd-product-tile__content">
-        <h3 class="fd-product-tile__title">Product Tile Button</h3>
+    <div tabindex="0" class="fd-tile fd-tile--launch fd-tile--s">
+        <div class="fd-tile__header">
+            <div class="fd-tile__title">Launch Tile (Logo) Title Text</div>
+            <div class="fd-tile__subtitle">Launch Tile Subtitle</div>
+        </div>
+        <div class="fd-tile__content">
+            <img class="fd-tile__logo" src="//unpkg.com/fundamental-styles/dist/images/sap-logo.png" srcset="//unpkg.com/fundamental-styles/dist/images/sap-logo@2x.png 1x, //unpkg.com/fundamental-styles/dist/images/sap-logo@3x.png 2x, //unpkg.com/fundamental-styles/dist/images/sap-logo@4x.png 3x" alt="SAP">
+        </div>
+        <div class="fd-tile__footer">
+            <span class="fd-tile__footer-text">Unit, Notification</span>
+        </div>
     </div>
 </div>
-
 {% endcapture %}
-{% include display-component.html component=tile %}
+{% include display-component.html component=tile-launch %}
 
 <br>
 
-## Disabled state
-Add class `is-disabled`
+## KPI Tile (Numeric Content)
+Key Performance Indicators (KPIs) are used to measure and monitor a company’s performance at a strategic and operational level.The number of digits is limited by the size of the tiles.
```

Comment:
The KPI Tile control supports a limited number of digits which varies depending on the size of the tile.